sku.config.js

```diff
@@ -1,3 +1,5 @@
+const isPullRequest = !!process.env.TRAVIS_PULL_REQUEST_SHA;
```

Comment:
This is true for dev as well though right? I would say its more doing something custom for gh-pages and the same for surge and dev.

Reply:
Note the double negative. Would this be clearer as `Boolean(process.env.TRAVIS_PULL_REQUEST_SHA)`? Is there a better way to write this?

Reply:
In dev, it's `undefined`, but in Travis it's `""`.

Reply:
Updated, let me know what you think 👍🏻